blocks/library/text/index.js

```diff
@@ -47,7 +47,25 @@ registerBlock( 'core/text', {
 		}
 	],
 
-	edit( { attributes, setAttributes, insertBlockAfter } ) {
+	formatting: [
```

Comment:
Just pinging @aduth maybe he have some thoughts on the block API part of this PR. The `formatting` attribute here and the two props `onFormatChange` and `formats`